app/cafe.py

```diff
@@ -0,0 +1,20 @@
+from datetime import date
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError, NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("Visitor is not vaccinated.")
+
+        if visitor["vaccine"]["expiration_date"] < date.today():
+            raise OutdatedVaccineError("Vaccine is outdated.")
+
+        if visitor["wearing_a_mask"] is False:
```

Comment:
Consider checking if the 'wearing_a_mask' key exists in the visitor dictionary before accessing it. This will prevent a potential KeyError if the key is missing.